src/main/java/com/vfasad/controller/reports/CommonReportController.java

```diff
@@ -136,9 +149,43 @@ public ModelAndView monthReport(@PathVariable int year,
         ordersByUserId.keySet()
                 .forEach(userId -> areaByUserId.put(userId, ordersByUserId.get(userId).stream().mapToDouble(this::getSumArea).sum()));
         modelAndView.addObject("areaByUserId", areaByUserId);
+        boolean reportIsOpened = (reportService.findByYearAndMonthReportCommon(year, month)).equals(Optional.empty());
+        modelAndView.addObject("reportIsOpened", reportIsOpened);
+        modelAndView.addObject("reportCommon", reportService.findByYearAndMonthReportCommon(year, month).orElse(null));
```

Comment:
Method findByYearAndMonthReportCommon is called twice. We have no performance requirements, but I'd propose to avoid so obvious overheads :)